source/use-cases/sensitive-data-encryption.txt

```diff
@@ -13,26 +13,65 @@ Sensitive Data Encryption
+
+This guide provides steps for setup and implementation of CSFLE.
+
+.. note::
+
+  Client-side Field Level Encryption is available starting in MongoDB 4.2 Enterprise only
 
 Problem
 ~~~~~~~
 
-Pellentesque non mi et est congue mattis pretium sit amet mi. Praesent
-nec tortor nec arcu molestie dapibus. Phasellus at nunc ut quam molestie
-finibus. Suspendisse finibus lectus eget nulla eleifend, vel consequat
-leo rhoncus. Phasellus lacinia tortor id aliquet condimentum. Morbi
-sodales est a leo rhoncus, non pretium neque dignissim.
+In this scenario, we secure sensitive data on a Medical Care Management System which stores patients' personal information, appointments, and vital sign measurement reports for a fictitious company, *MedcoMD*. None of the patient data is public, and certain data such as their social security number (SSN) and vital sign measurements are particularly sensitive. It is important for the company and the patient that the data is kept private and secure.
+
+MedcoMD needs this system to satisfy the following use cases:
+
+- Doctors use the system to access Patients' medical records and add new vital sign measurements.
+- Receptionists use the system to schedule appointments for Patients, who are identified by a combination of contact information and their SSN.
+- Receptionists cannot access nor update Patients' medical records.
+
+MedcoMD is also concerned with disclosure of private information through any of the following methods:
+
+- Printing the information or leaving it displayed on a screen where unauthorized people can view it
+- Communicating on an insecure network
+- Gaining access to the database on one of the permissioned users
+
+What can MedcoMD do to balance the functionality and access restrictions of their Medical Care Management System?
 
 Solution
 ~~~~~~~~
 
-Praesent sit amet elit pretium mi molestie porta a sit amet ipsum. Sed
-sollicitudin purus et mi blandit sodales.
+The MedcoMD software engineers review the Medical Care Management System specification and determine that Doctors and Receptionists require different levels of access to the data.
+
 
-- Deterministic vs Non-Deterministic
+While role-based access control could separate the access, the granularity is only as fine as a collection
+
+
+In order to minimize the chance of disclosure of sensitive data, they decide encryption is the best option
+
+
+The MedcoMD developers decide this is an ideal opportunity to use CSFLE which allows them to select specific fields of a document that should be kept encrypted on the server. This feature will perform validation and encryption of the specified sensitive data fields, ensuring that they remain encrypted on the server. 
+
+be provided with a client that can access and
+ and the Receptionists will be provided
```

Comment:
Looks like this spot is where we want to describe "doctor client vs receptionist client"?

Reply:
Updated to explain this.